docs/src/examples/collections/Table/Variations/TableExampleSortable.js

```diff
@@ -1,62 +1,51 @@
 import _ from 'lodash'
-import React, { Component } from 'react'
-import { Table } from 'semantic-ui-react'
+import React, { useState } from "react";
+import { Header, Table } from "semantic-ui-react";
 
 const tableData = [
-  { name: 'John', age: 15, gender: 'Male' },
-  { name: 'Amber', age: 40, gender: 'Female' },
-  { name: 'Leslie', age: 25, gender: 'Other' },
-  { name: 'Ben', age: 70, gender: 'Male' },
-]
+  { name: "John", age: 15, gender: "Male" },
+  { name: "Amber", age: 40, gender: "Female" },
+  { name: "Leslie", age: 25, gender: "Other" },
+  { name: "Ben", age: 70, gender: "Male" },
+];
 
-export default class TableExampleSortable extends Component {
-  state = {
-    column: null,
-    data: tableData,
-    direction: null,
-  }
-
-  handleSort = (clickedColumn) => () => {
-    const { column, data, direction } = this.state
+function Table() {
```

Comment:
```suggestion
export default TableExampleSortable() {
```

- default export works for functions 😸 
- let's keep existing example name